Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkBasic/EyelinkBasicLegacy/EyelinkClearCalDisplay.m

```diff
@@ -0,0 +1,4 @@
+function EyelinkClearCalDisplay(el)
+    warning('EyelinkToolbox:LegacyClearCalDisplay',['The function EyelinkClearCalDisplay() is deprecated. Please update ', ...
+        'your script to use the current method for handling camera setup mode callbacks with PsychEyelinkDispatchCallback.m.']);
+	warning('off', 'EyelinkToolbox:LegacyClearCalDisplay');
```

Comment:
What happened to the actual clearing code that used to be in here?

Reply:
resolved in 33984c0ef6fc9d2a3dc40f30ab9710ad031d9938